### ISSUE TITLE
Workshop in Opf DS Meetup - Changes and Tasks to be done

### ISSUE THREAD

**Issue by aakankshaduggal** [aicoe-aiops/ocp-ci-analysis#583]
@aakankshaduggal and @suppathak are presenting the Workshop - Build AIOps tools using Operate First in the Operate First Data Science Community Meetup on August 9th. Here are a few tasks that need to be done before that to ensure a smooth session.

Before the meetup -

- [x] Update the meetup agenda and send out pre-requisites to the meetup attendees to collect the Github handles [here.](https://etherpad.opendev.org/p/workshop_scale_2022)
- [x] Sync up with @HumairAK and ensure everyone has access to the Operate first services 
- [x] Test out all services and check if all notebooks are running as expected
- [x] Create a new bitwarden link with env credentials
- [x] Update the JH image for this repository with the latest changes to avoid forking the repo part 
- [x] Send out a reminder email on Monday with the bitwarden link, creating env instructions, and how to create a Github access token documentation.

Presentation in the meetup - This workshop will be a shorter version of the original workshop due to a time limit on the meetup. The maximum we could is 45 mins. Here's the new plan for the workshop and the following services are reqd to be running smoothly -

- [x] Start the demo by spawning up a Jupyterhub image for this repo, ideally everyone should have the env file ready before the workshop,
- [x] Dive into the jupyter notebooks for model development. Quick overview and mainly just re-running the notebooks.
- [x] Cloudbeaver introduction and show the tables created
- [x] Superset demo and a tutorial of how we can create our own charts and dashboards
